pkg/cli/klothomain.go

```diff
@@ -24,14 +24,12 @@ import (
 )
 
 type KlothoMain struct {
-	UpdateStream     string
-	Version          string
-	VersionQualifier string
-	PluginSetup      func(*PluginSetBuilder) error
+	DefaultUpdateStream string
```

Comment:
nit: Is this the default if they have set it in their config?

Reply:
I think so? It's the default, which they can then override in their options. (It's actually not overridable on a one-invocation-only basis, only via persisted options.)

(As an aside: I'm trying to use "options" for the CLI-wide stuff [which is currently only the update stream], to differentiate it from the klotho project configs `-c`.)